CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## 2.8.1
 * Fix support for using multiple sparkly sessions during tests
 * SparklySession does not persist modifications to os.environ
+* Support ElasticSearch 7+ by making `type` optional.
```

Comment:
Let's just say 7, as `7+` contains 8 etc., and we don't know if they'll have other breaking changes. Let's also just say `type` without enclosing it in \`, since written that way it usually refers to a variable or an argument named like that, and in our case the argument is `es_type`.